domains/dhananjai.json

```diff
@@ -0,0 +1,11 @@
+{ 
+  "description": "My portflio website",
+  "repo": "https://github.com/Kuro0911/my-site",
+  "owner": {
+    "username": "Kuro0911",
+    "email": "dhananjaisharma09@gmail.com"
+  },
+  "record": {
+    "CNAME": "kuro0911.github.io/my-site"
```

Comment:
`CNAME` records do not support `/` in them.
Please read the [documentation](https://docs.is-a.dev/github_pages/) to learn how to setup GitHub Pages with is-a.dev subdomain.